pytest.ini

```diff
@@ -0,0 +1,10 @@
+[pytest]
+addopts =
+    # show summary of all tests that did not pass
+    -ra
+    # Make tracebacks shorter
+    --tb=short
```

Comment:
pytorch uses native. I'm not too familiar with it, do you have a preference?

Reply:
Depends. `native` might get better tracebacks for non-python errors. `short` is even more concise for default Python tracebacks. No strong opinion.